docs/tsconfig/it/options/alwaysStrict.md

```diff
@@ -0,0 +1,8 @@
+---
+display: "Sempre Rigoroso"
+oneline: "Assicura che "use strict" sia sempre inserito all'inizio del file"
```

Comment:
This use of `"` in a string is probably what's breaking CI

Reply:
> This use of `"` in a quote string is probably what's breaking CI
Ok thank you, later Im going to review the code and try again.